### DISCUSSION TITLE
core目录下多余的pybind11 namespace ？

### DISCUSSION THREAD

**Discussion by lloydzhou** [TuGraph-family/tugraph-db#149] [Q&A] [answered]
@knightast 您好

最近在看代码的时候发现core目录下面的几个头文件有申明`pybind11`和`python`的`namespace`，这个是有什么特殊的意义吗？我发现把这几个移除之后并不影响编译和跑测试用例。

在以下四个文件内部
src/core/edge_index.h
src/core/graph_edge_iterator.h
src/core/graph_vertex_iterator.h
src/core/vertex_index.h

**Comment by knightast** [comment, marked as answer]
应该是之前调python plugin的时候留下的，目测没啥用了，可以提个pr删了 :)

**Reply by lloydzhou** [reply]
尝试删除，但是代码跑`github ci`的时候出现了`core dumped`
![image](https://user-images.githubusercontent.com/1826685/212549830-111c2924-98f4-46db-a49d-4a8c747f666e.png)
